src/lib/ast.rs

```diff
@@ -180,7 +197,7 @@ mod test {
     fn test_invalid_start_rule(){
         let mut grm = GrammarAST::new();
         grm.start = Some("A".to_string());
-        grm.add_prod("B".to_string(), vec!());
+        grm.add_prod("B".to_string(), vec!(), None);
```

Comment:
Not really. http://xion.io/post/code/rust-optional-args.html is the best that I know of.